src/App.js

```diff
@@ -1,24 +1,86 @@
-import logo from './logo.svg';
+import { useState, useEffect } from 'react';
+import { getUser, logout } from './services/fetch-utils';
+import { 
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+  Link,
+} from 'react-router-dom';
+import AuthPage from './AuthPage';
+import CharacterDetails from './CharacterDetails/CharacterDetails';
+import CharacterList from './CharacterList/CharacterList';
+import CreateCharacter from './CreateCharacter/CreateCharacter';
 import './App.css';
 
 function App() {
+  const [email, setEmail] = useState('');
+  const [token, setToken] = useState('');
+
+  useEffect(() =>{
+    const user = getUser();
+    if (user) {
+      setToken(user.access_token);
+      setEmail(user.user.email);
+    }
+  }, []);
+
+  async function handleLogout(){
+    await logout();
+    setEmail('');
+    setToken('');
+  }
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <header className="App-header">
+          <h2> Currently Logged in as {email}</h2>
+          <nav>
+            <ul>
+              <li>
+                <Link to='/'>Characters</Link>
+              </li>
+              <li>
+                <Link to="/create-character">Create A Character</Link>
+              </li>
+            </ul>
+          </nav>
+          <button onClick={handleLogout}>Log Out</button>
```

Comment:
lets make this so the header conditionally renders if a user is logged in, maybe using the `&&` operator.